debian/libglib2.0-0.symbols

```diff
@@ -582,6 +582,7 @@ libgio-2.0.so.0 libglib2.0-0 #MINVER#
  g_desktop_app_info_has_key@Base 2.35.8
  g_desktop_app_info_launch_action@Base 2.37.5
  g_desktop_app_info_launch_uris_as_manager@Base 2.28.0
+ g_desktop_app_info_launch_uris_as_manager_with_fds@Base 2.28.0
```

Comment:
Nitpick: The version here is wrong; this should be the version number of the package you’re about to produce, if we want the dependencies to be right. However, since everything will be using the new GLib package in EOS, it doesn’t matter if the dependencies are wrong, so this is just a nitpick.